src/concepts/pipeline/filtering.md

```diff
@@ -0,0 +1,105 @@
+
+What: Causes data to be written to the error stream or to be dropped entirely.
+
+When: Filter out data we absolutely know we will never need to see (e.g. data from forked applications).
+
+How: Can filter out namespaces, doctypes, or URIs currently; in the extreme can filter on any [message Attribute](https://github.com/mozilla/gcp-ingestion/blob/master/ingestion-core/src/main/java/com/mozilla/telemetry/ingestion/core/Constant.java#L8) or payload field.
+
+### JSON Schema Filtering
+
+Where: During ingestion, as defined in the [payload schema](https://github.com/mozilla-services/mozilla-pipeline-schemas/).
+
+What: Causes data to be written to the error stream.
+
+When: When trying to remove bad analysis data that we know we will never need (e.g. huge values, improper strings, etc.). Usually these indicate something went wrong with the payload.
+
+How: Can filter on values in the payload, using the [JSON schema](https://json-schema.org/understanding-json-schema/).
+
+### Filtering from the stable tables
+
+Where: After ingestion to live tables, but [before copying to the stable tables](https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/copy_deduplicate.py#L40).
+
+What: Allows data to exist in the live tables, but removes it from the stable tables.
+
+When: Use for data that may be needed for some analyses on recent data, but not for data that will need long-term historical analyses or for use in any downstream reporting. For example, we filter out pings from automation (e.g. CI) here, so that analysis is unaffected by them, but we can still analyze what the recent CI data looks like. We also drop duplicate pings (per the document-id).
+
+How: Can filter on any field in the schema, or any metadata.
+
+### Filtering from the exposed views
+
+Where: After ingestion to stable tables ([example](https://github.com/mozilla/bigquery-etl/blob/master/sql/moz-fx-data-shared-prod/telemetry/lockwise_mobile_events_v1/view.sql#L17)).
+
+What: Allows data to exist in stable tables, but not be exposed to users when accessing views.
+
+When: Use for data that is a footgun for end-users (e.g. data that was collected before a product was launched), but will probably be needed by data science or eng.
+
+How: Can filter on any field in the schema, or any metadata.
+
+### Optional filtering in Looker Explores
+
+Where: In the explore, Looker creates a [default filter for a field](https://docs.looker.com/reference/field-params/default_value).
+
+What: Allows data to exist in views, and optionally allows users to query that data (but not by default).
+
+When: Use this for data that most of the time should not be queried in Looker. Downside is too many of these will clutter the Looker explore.
+
+How: Can filter on any field in the schema, or any metadata.
+
+## Querying the Error Stream
```

Comment:
Gonna mention the aggregated error counts dataset as well?

Reply:
The dashboard below uses that table, is that sufficient?